ansible/playbook.yml

```diff
@@ -0,0 +1,163 @@
+- become: true
+  hosts: all
+  name: Impostor-install
+  gather_facts: true
+  vars:
+    impostor_nightly_zip_path: "/home/ec2-user/Impostor-Server-latest.zip"
+    impostor_path: "/home/ec2-user/Impostor"
+    impostor_plugins: "/home/ec2-user/Impostor/Plugins"
+    impostor_user: "ec2-user"
+    public_port: "443"
```

Comment:
```
"Server": {
    "PublicIp": "34.232.46.194",
    "PublicPort": 22023,
    "ListenIp": "0.0.0.0",
    "ListenPort": 22023
  },
```

That is my current config. The Public IP will need changed to a domain for a NLB, but then the public and listen ports will need to be 22023. This is because during match making the Impostor.Http will read the PublicIP and port as what UDP endpoint to give to the client to tell it where to connect to.

Reply:
Should we just hardcode your public IP or FQDN into our config, or did we want to keep the playbook more generic so it could be used elsewhere?

Reply:
Actually, looking over your other PR, I see you are setting "'matchmaker_domain=${local.matchmaker_fqdn}' -e 'game_domain=${local.gameserver_fqdn}'" in the Ansible provisioner. Which one of these goes where? I'm not 100% yet on what exactly the matchmaker is doing or how it works. I assume "PublicIP" in the config.json should be "gameserver_fqdn", right?